### DISCUSSION TITLE
Download entire folder

### DISCUSSION THREAD

**Discussion by AQSAMA** [ente-io/ente#914] [Enhancements]
I desperately want to download an entire folder, or several folders, and when they are downloaded, each folder will have the same name that I uploaded, and the photos will also keep their history

And please add the feature of selecting all in the folder and counting the number of photos

**Comment by oiao** [comment]
I second this feature. 
The related issue #913 has been closed and moved to a discussion -- Is there a way to know if and when this will be picked up @vishnukvmd?

**Reply by vishnukvmd** [reply]
Issues are bugs, discussions for feature requests. Please follow this discussion to track progress.

**Reply by oiao** [reply]
Thanks a lot for the clarification @vishnukvmd. Given that this was opened two years ago, could you perhaps also comment on my initial question -- Is there a way to know if and when this feature request will be picked up?
